openshift_tools/ircbot/openshift_sre/openshift_sre.py

```diff
@@ -341,9 +342,8 @@ def monitor_weekend(bot, trigger):
     """Reminds users that channels are un-monitored on weekends. Will not trigger more than once every 10 minutes."""
     if trigger.sender in MONITORED_CHANNELS and dt.utcnow().weekday() > 4:
         leads = get_shift_leads(bot, trigger.sender)
-        bot.reply('This channel is unmonitored on weekends. If you have a sev 1, submit a SNOW ticket to page the '
-                  'on-call SRE ({oncall}). Submit your SNOW ticket at {url}'.format(oncall=leads[ONCALL['name']],
-                                                                                    url=SNOW_URL))
+        bot.reply('This channel is unmonitored on weekends. See {url} for Engineer Escalations'.format(
+            oncall=leads[ONCALL['name']], url=ESCALATION_URL))
```

Comment:
pylint is rightly complaining that you're not using 'oncall' in the string.

Reply:
Clearly, thanks! I glossed right over that!